src/main/java/com/three_tech_solutions/slot_app/exceptions/responses/ApiError.java

```diff
@@ -9,8 +9,8 @@
 public record ApiError(
         int status,
         String errorMessage,
+        List<String> errors,
```

Comment:
Las APIs deberían ser claras con la información que devuelven y acá estamos devolviendo dos atributos diferentes para manejar los mensajes de error. Si soy el cliente que consume la API, ¿cuál debería chequear? ¿errorMessage, errors o ambas? Es mucho mejor si la API retorna en único campo el mensaje de error, ya sea en una lista porque son varios errores o en unico mensaje.

En este caso me gusta que en las validaciones hayas puesto la lista de todos los posibles errores que haya hecho el cliente al hacer la requests, sin embargo, al tener ambos atributos hay algunos casos que pasa lo siguiente

<img width="678" height="416" alt="Image" src="https://github.com/user-attachments/assets/948248b2-efb2-4f4d-81bf-78f684ab10c5" />

Se devuelve un `errorMessage` pero el `errors` es null, es medio confuso. 

Dejaria esto asi


```suggestion
        List<String> errors,
```

y donde se utilizaba el errorMessage, guardaria el error para que este en una lista